docs/api-reference/v4/endpoints/event-actions.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "Apply Action to Events"
+api: "POST https://api.flare.io/firework/v4/events/actions/{action}"
```

Comment:
I thought we said the action would be in the payload?

Reply:
Discussion is ongoing in the [RFC](https://flareio.atlassian.net/wiki/spaces/engineering/pages/609222671/2024-12-17+-+Remediate+and+ignore+activities+in+the+Public+API). Will update the PR when the final solution is agreed upon.